content/events/2019/11/2019-11-19-jez-humble-building-scaling-high-performing-technology-organizations.md

```diff
@@ -1,24 +1,27 @@
 ---
 slug: jez-humble-building-scaling-high-performing-technology-organizations
-title: 'Jez Humble&#58; Building and Scaling High Performing Technology Organizations'
-summary: 'This talk presents the findings of that research, including how to measure the performance of software teams, what capabilities organizations should invest in to drive higher performance, and how software leaders can apply these findings in their own organizations&#46;
-'
-featured_image: 
-  uid: 
+title: "Jez Humble&#58; Building and Scaling High Performing Technology Organizations"
+summary: "This talk presents the findings of that research, including how to measure the performance of software teams, what capabilities organizations should invest in to drive higher performance, and how software leaders can apply these findings in their own organizations&#46;"
+featured_image:
+  uid: devops-jez-humble
   alt: ''
-event_type: 
+event_type:
   - zoom
 date: 2019-11-19 13:00:00 -0500
 end_date: 2019-11-19 14:00:00 -0500
 event_organizer: DigitalGov University
-host: DevOps COP 
+host: DevOps Community
 registration_url: https://www.eventbrite.com/e/jez-humble-building-and-scaling-high-performing-technology-organizations-registration-80579640875
 
 ---
 
+{{< img src="devops-jez-humble" >}}
+
 How can we apply technology to drive business value? For years, we've been told that the performance of software delivery teams doesn't matter―that it can't provide a competitive advantage to our companies.
```

Comment:
How can we apply technology to drive business value in organizations? For years, we've been told that the performance of software delivery teams doesn't matter―that it can't provide a competitive advantage to our companies, or better outcomes in the public sector.

Reply:
Done. Added a few minor copy edits.